packages/react/src/modules/slack/components/SlackChannelCombobox/SlackChannelCombobox.tsx

```diff
@@ -83,6 +90,16 @@ export const SlackChannelCombobox = ({
     SlackChannelConnection[] | null
   >(null);
 
+  useEffect(() => {
+    if (comboboxListOpen) {
+      // Timeout to allow for the state to update and the component to re-render
+      // when we change the `comboboxListOpen` state upon focus
+      setTimeout(() => {
+        document.getElementById("slack-channel-search")?.focus();
```

Comment:
I think here it might be best to use:

`comboboxRef.current.querySelector('.rsk-combobox__searchbar__input')?.focus()` so you don't need the id

Reply:
is it better to not use the id? the id will exist regardless of this - the reason I'm asking is because i'm getting a typescript error about the `comboboxRef.current` possibly being a never type

Reply:
Checking if `ref.current` is defined may fix the typescript error